server/game/cards/attachments/04/motherofdragons.js

```diff
@@ -0,0 +1,48 @@
+const _ = require('underscore');
+const DrawCard = require('../../../drawcard.js');
+
+class MotherOfDragons extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
```

Comment:
As this is a 'Challenges Action' effect, this would be better implemented as an action so that the player has a menu and can trigger the effect when they want, rather than in reaction to attackers/defenders.  It will also simplify the implementation a bit